core/species.cpp

```diff
@@ -9812,13 +9817,12 @@ void Species::__CheckNodePedigreeIDs(__attribute__((unused)) EidosInterpreter *p
 	// could be nodes in the node table with haplosome pedigree IDs greater than those in use by individuals, in nonWF models.
 	// See https://github.com/MesserLab/SLiM/pull/420 for an example model that does this very easily.
 	
-	// Also, check for duplicate pedigree IDs, just in case.  __CreateSubpopulationsFromTabulation() does this for individual
-	// pedigree IDs; we do it for node pedigree IDs.  I decided to use a vector with std::sort() to check even though it is
-	// O(n log n), rather than a hash table for O(n), because I'm nervous about hitting a bad edge case with the hash table
-	// due to the nature of the values being inserted.  Shouldn't be a big deal in the grand scheme of things.
+	// Previously, we checked for duplicate pedigree IDs here as well, just in case.
+    // __CreateSubpopulationsFromTabulation() does this for haplosome IDs in
+    // living individuals already pedigree IDs; however, it was found to be 
```

Comment:
the comment text here is garbled and I'm not sure what it is intended to say; please fix

Reply:
whoops; done